examples/research_projects/xtreme-s/README.md

```diff
@@ -81,9 +81,9 @@ The following command shows how to fine-tune the [XLS-R](https://huggingface.co/
 python -m torch.distributed.launch \
     --nproc_per_node=8 \
     run_xtreme_s.py \
+    --task="mls" \
+    --language="all" \
     --model_name_or_path="facebook/wav2vec2-xls-r-300m" \
-    --dataset_name="google/xtreme_s" \
```

Comment:
The script is called `run_xtreme_s.py` so we don't need to specific this here again